package.json

```diff
@@ -24,7 +24,7 @@
     "url": "https://github.com/mixmaxhq/rollup-plugin-handlebars-plus/issues"
   },
   "homepage": "https://github.com/mixmaxhq/rollup-plugin-handlebars-plus#readme",
-  "dependencies": {
-    "handlebars": "^4.0.5"
+  "peerDependencies": {
+    "handlebars": ">=3.0.0 < 5"
```

Comment:
Just checking that dropping the version requirement to 3.0.0 doesn't require we make any changes to how we use Handlebars throughout the library?

Reply:
I checked your module from version 1.3.0 to 4. I got an error when version was smaller than 3. It caused by this line https://github.com/mixmaxhq/rollup-plugin-handlebars-plus/blob/master/src/index.js#L61
As I see, accept should not be called under handlebars version 3, but everything works with and above 3.0.0.
So I thought I should add supported version from 3.0.0.
Maybe if you have time, you can modify the plugin to be able to precompile handlebars 1.3.0 as well. It should not be a big work the only problem is this accept call. If I commented it out, I got the template without any problem.